targets/lantiq-xrx200

```diff
@@ -1,3 +1,12 @@
 device('avm-fritz-box-7412', 'avm_fritz7412', {
         factory = false,
 })
+
+-- TP-Link
+
+device('tp-link-td-w8970', 'tplink_tdw8970')
```

Comment:
This currently still needs a BROKEN flag, as we have not yet made a decision on how to handle hybrid LAN/WAN ports.

Reply:
Done :)